src/utils/arrayDifference.ts

```diff
@@ -0,0 +1,5 @@
+function arrayDifference<TItem>(array1: TItem[], array2: TItem[]): TItem[] {
```

Comment:
Can we document this function? What exactly is it doing? Is it returning the any values that are either in array1 or array2 but not in both? Or something else?

Reply:
Function documented, let me know what you think! Regarding the other question, it's because some object keys don't fit the @typescript-eslint/naming-convention rule, that expects them to be camelCase, UPPER_CASE or PascalCase.

Reply:
Got it, thanks!